modules/nw-creating-anp-egress-policy.adoc

```diff
@@ -0,0 +1,203 @@
+
+* An IP address range in CIDR format
+
+* A DNS name that resolves to an IP address
+
+* A port number
+
+* A protocol that is one of the following protocols: TCP, UDP, and SCTP
+
+[NOTE]
+====
+`Nodes` and `networks` peer only work for egress traffic.
+====
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with administrator privileges for the project.
+* You installed the `oc` CLI.
+
+.Procedure
+
+. Create a YAML file to define the egress `AdminNetworkPolicy` CR using nodes peer:
++
+[source,terminal]
+----
+oc create -f <policy_name>.yaml
+----
++
+
+. Define the egress policy in the YAML file you created based on the following example:
++
+.Example of ANP egress policy using nodes peer
+[%collapsible]
+====
+[source,terminal]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: node-as-egress-peer
+spec:
+  priority: 30 <1>
+  subject:
+    namespaces:
+      matchLabels:
+        tenant: backend-storage <2>
+  egress:
+    - name: "allow_egress" <3>
+      action: "Allow" <4>
+      to:
+        - nodes: <5>
+            matchExpressions: <6>
+              - key: team
+                operator: In
+                values:
+                  - ocp
+      ports: <7>
+        - portNumber:
+            protocol: TCP
+            port: 27017
+        - portNumber:
+            port: 8080
+            protocol: TCP
+    - name: "deny egress"
+      action: "Deny"
+      to:
+        - nodes:
+            matchExpressions:
+              - key: team
+                operator: In
+                values:
+                  - qe
+    - name: "pass egress"
+      action: "Pass"
+      to:
+        - nodes:
+            matchExpressions:
+              - key: tenant
+                operator: Exists
+----
+<1> Specifies the priority of the ANP. Accepts a maximum of 100 ANP with values 0-99. Creating an ANP with the same priority is supported for non-overlapping policies. Administrators should make sure that ANPs created at the same priority are not overlapping because it is non-deterministic as to which policy will take effect.
+<2> Specifies all of the pods owned by the tenant `backend-storage`.
+<3> Specifies the name of the ANP egress policy.
+<4> Specifies the `action` taken for the first rule in the egress policy. `Action` field accepts values of `Allow`, `Deny`, or `Pass`. In the example, `team ocp` is allowed to talk to `backend-storage` and `team qe` cannot talk to `backend-storage`.
+<5> `Nodes` peer accepts `matchExpressions` and `networks` peer accepts a CIDR range.
+<6> Specifies the nodes to apply the policy to through `matchExpressions` labels.
+<7> Specifies the ports to `Allow` egress traffic from `team ocp` to `backend-storage`.
+====
++
+. Apply the policy by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <policy_name>.yaml
```

Comment:
If it is created we do not apply again.
oc create or oc apply are used

oc replace in case we want to update it with YAML file